package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
+    "izitoast": "^1.4.0",
```

Comment:
Do we need to vendorize this library?

Reply:
we use this for our toastr notifications on successful or unsuccessful login, so we don't need it, but I think it is quite nice

Reply:
Not questioning the use of the library. What I am questioning is why we have it in the package.jsno and at the same time we add the minified JavaScript and css of it to our code.

I prefer not to vendorize anything

Reply:
oh, right, sorry I misunderstood, and we totally can remove that since we are importing it from our assets directory

Reply:
Is there a way to use the package.json version instead of the assets folder one?